### PR TITLE
fix(PPPLMER-59482) DE - Translation change for Powered by Paypal

### PR DESCRIPTION
**Pull request by rupeshdabbir**
- Translation for 'DE' to Powered by PayPal
- Please check the ticket for Approval for the content.

### CODE REVIEW

**Comment by rupeshdabbir** [comment]
@khanghoang ^^

File: src/button/template/content.json [hunk 1]
```diff
@@ -1220,7 +1220,7 @@
             "later_tag": "Jetzt bei uns bequem in Raten zahlen.",
             "pay": "Mit {logo:paypal} zahlen",
             "buynow": "{logo:pp} {logo:paypal} Jetzt kaufen",
-            "poweredBy": "Abgewickelt durch {logo:paypal}"
+            "poweredBy": "Powered by PayPal"
```
Comment (bluepnume) [inline src/button/template/content.json]: Not clear on why we're putting english in the DE translations?

Comment (bluepnume) [reply src/button/template/content.json]: Also either way we should keep `{logo:paypal}`

Comment (rupeshdabbir) [reply src/button/template/content.json]: I had the same Question but in the ticket `PPPLMER-59482` product wanted english in 'DE' translation ( I know .. I know...)

Comment (rupeshdabbir) [reply src/button/template/content.json]: I updated the code Daniel, could you check when you get time?

**Pull request merged**